masschallenge/impact-api
# [AC-5345] Winner Statuses should be available in Salesforce

**STEP 2: This step creates the history event.  The creation of the Startup Roles and the assignment of them to the winners happens in STEP 1 on accelerate.**

**To Test On The Branch**
- Run `make build; make dev`.
- In a separate tab, run `make load-db GZ_FILE=../accelerate/db_cache/initial_schema.sql.gz`.
- Run `make grant-permissions PERMISSION_USER=demoadmin@masschallenge.org PERMISSION_CLASSES=v1_clients`.
- In the browser, navigate to [http://localhost:8000/api/v1/organization/1707/history/](http://localhost:8000/api/v1/organization/1707/history/).
- Check that the `became winner` event shows for this user and that the datetime is the same as the end_date for the program (the startup status doesn't have a credible created_at date).
- You can use the list of startups from accelerate to test other organization's `became winner` event.
